stubs/webapp2/__init__.pyi

```diff
@@ -1,10 +1,18 @@
 import datetime
-from typing import Any, Dict, List, Optional, Tuple, Union
+from typing import Any, Callable, Dict, List, Optional, Tuple, Union, Pattern
```

Comment:
Ahh, I think we should not do this, reason:
1. `Any` is used heavily in stubs file, you can pick any of the file -- finding reasons for all of them is cumbersome work ( or unnecessary in some cases because someone has already added Any for a reason without mentioning the reason ).
2. In somecases, libraries do have thier official stubs and we have to wrtie stubs for ourselves which forces us to use Any in some cases and finding reason for them is also a time taking.

Reply:
1. I see.
2. Not sure what you mean by this, can you elaborate?

Reply:
2. In some cases libraries do not have official stubs and in those cases, we have to write stubs by ourselves which forces us to use Any in some method/function signatures. And finding reasons for them is also a taking process.